hs-init.hs

```diff
@@ -442,14 +448,14 @@ reset = setSGR [Reset]
 prompt :: IO Text
 prompt = do
   setColor Blue
-  T.putStr "  →   "
+  putStrFlush " ->   "
```

Comment:
Can I ask you to make here 2 spaces before the arrow, I think it adds a little bit more readability :slightly_smiling_face: